docs/conf.py

```diff
@@ -92,8 +101,27 @@
 plot_html_show_formats = False
 plot_formats = ["svg"]
 
-# Redirects for pages that have moved
-redirects = {}
+# ----------------------------------------------------------------------------------
+# Redirects
+# ----------------------------------------------------------------------------------
+
+_inlined_apis = [
+    ("qiskit_addon_aqc_tensor.objective", "OneMinusFidelity"),
+    ("qiskit_addon_aqc_tensor.simulation", "TensorNetworkState"),
+    ("qiskit_addon_aqc_tensor.simulation", "TensorNetworkSimulationSettings"),
+    ("qiskit_addon_aqc_tensor.simulation", "tensornetwork_from_circuit"),
+    ("qiskit_addon_aqc_tensor.simulation", "apply_circuit_to_state"),
+    ("qiskit_addon_aqc_tensor.simulation", "compute_overlap"),
+]
+
+redirects = {
+    "stubs/qiskit_addon_aqc_tensor.ansatz_generation.generate_ansatz_from_circuit": "../apidocs/qiskit-addon-aqc-tensor.html#qiskit_addon_aqc_tensor.generate_ansatz_from_circuit",
+    **{
+        f"stubs/{module}.{name}": f"../apidocs/{module.split('.')[-1]}.html#{module}.{name}"
```

Comment:
Calling `module.split('.')[-1]` is new because this repo doesn't put the full module path in the file names in `apidocs` - that is fine with me because qiskit/documentation stripped it out anyways.